hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/table/BaseEntityEventType.java

```diff
@@ -13,5 +13,5 @@
  *
  */
 public enum BaseEntityEventType {
-    NEW_ENTITY, UPDATED_ENTITY, DELETE_ENTITY, SELECTED_ENTITY, MAXIMIZED, MINIMIZED;
+    ADD_ENTITY, REMOVE_ENTITIES, UPDATED_ENTITY, SELECTED_ENTITY, MAXIMIZED, MINIMIZED;
```

Comment:
`REMOVE_ENTITIES` is the only constants in plural. Why it differs from the others?